pyproject.toml

```diff
@@ -44,6 +44,8 @@ flake8-import-order = "^0.18.1"
 pytest              = "^7.2.0"
 pytest-asyncio      = "^0.20.1"
 mypy                = "^0.982"
+pandas              = "^1.5.3"
```

Comment:
I'm not sure how I feel about including pandas (even as a "dev" dependency) when it's only used in an example. My preference would be to tell mypy to ignore it completely and then revert these changes.
We can add

```
[mypy-pandas.*]
ignore_missing_imports = True
```
to the `mypy.ini` file.

Reply:
I made this change, but I'm still getting a type error in the example file, for the argument passed to read_csv.
Right now, I have the pandas lines commented out... which is not unreasonable, because if you run the example as-is you get an error because the iterator is already exhausted by the "save to file" part of the example.
If we don't like it commented out, though, I can look into the error more later, since I need to help with testing today.

Reply:
I'm fine with having that part of the example commented out. I think it's pretty common to see stuff like that in code snippets.